README.md

```diff
@@ -1,23 +1,24 @@
 # Packer
 
 [![Build Status][circleci-badge]][circleci]
-[![Windows Build Status][appveyor-badge]][appveyor]
+[![Discuss](https://img.shields.io/badge/discuss-packer-3d89ff?style=flat)](https://discuss.hashicorp.com/c/packer)
 [![PkgGoDev](https://pkg.go.dev/badge/github.com/hashicorp/packer)](https://pkg.go.dev/github.com/hashicorp/packer)
 [![GoReportCard][report-badge]][report]
 [![codecov](https://codecov.io/gh/hashicorp/packer/branch/master/graph/badge.svg)](https://codecov.io/gh/hashicorp/packer)
 
 [circleci-badge]: https://circleci.com/gh/hashicorp/packer.svg?style=svg
 [circleci]: https://app.circleci.com/pipelines/github/hashicorp/packer
 [appveyor-badge]: https://ci.appveyor.com/api/projects/status/miavlgnp989e5obc/branch/master?svg=true
-[appveyor]: https://ci.appveyor.com/project/hashicorp/packer
 [godoc-badge]: https://godoc.org/github.com/hashicorp/packer?status.svg
-[godoc]: https://godoc.org/github.com/hashicorp/packer
-[report-badge]: https://goreportcard.com/badge/github.com/hashicorp/packer
+[godoc]: https://godoc.org/github.com/hashicorp/packer	
+[report-badge]: https://goreportcard.com/badge/github.com/hashicorp/packer	
 [report]: https://goreportcard.com/report/github.com/hashicorp/packer
 
-* Website: https://www.packer.io
-* IRC: `#packer-tool` on Freenode
-* Mailing list: [Google Groups](https://groups.google.com/forum/#!forum/packer-tool)
+<p align="center" style="text-align:center;">
+  <a href="https://www.packer.io">
+    <img alt="HashiCorp Packer logo" src="website/public/img/logo-hashicorp.svg" width="500" />
+  </a>
+</p>
```

Comment:
This logo almost disapears on dark mode:

![Screenshot 2021-03-02 at 11 54 52](https://user-images.githubusercontent.com/1024852/109638325-31512880-7b4e-11eb-9ed7-cabc6b830cae.png)

Should we do something about that ?

Reply:
🤦🏼  oh no! We don't have the white version of this same logo. Right now there's no way to fix it.

Reply:
It looks so good on light mode 😭

Reply:
https://github.com/drcmda/dark-mode-github-readme-logos